include/kernel/thread.h

```diff
@@ -309,3 +285,28 @@ void awaken(void);
 void postpone(sched_context_t *sc);
 #endif
 
+/* Update the kernels timestamp and stores in ksCurTime.
```

Comment:
```suggestion
/* Update the kernel's timestamp and stores it in ksCurTime.
```

Reply:
Agreed, good moment for this. I also fixed the ksConsumed comment.